client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FormButton from 'components/forms/form-button';
+import StepHeader from 'signup/step-header';
+import Button from 'components/button';
+import { abtest } from 'lib/abtest';
+
+export default React.createClass( {
+	displayName: 'BluehostStoreStep',
+
+	redirectToPartner() {
+		if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
+			window.location.href = 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce';
+		} else {
+			window.location.href = 'https://www.bluehost.com/wordpress';
+		}
```

Comment:
Why not just a `href` and treat it as the link that it truly is? This is also better semantics, preserves the ability to open in a new tab, and is supported by the underlying [`<Button>` component](https://github.com/Automattic/wp-calypso/blob/f3a0ebdd46d1853a4fb47cc06ee49dc58968052e/client/components/button/index.jsx#L21). Speaking of, was there a reason for using `<FormButton />` vs. `<Button>`?